scripts/serve.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import SimpleHTTPServer
+
+class MyHTTPRequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+    def end_headers(self):
+        self.send_my_headers()
+        SimpleHTTPServer.SimpleHTTPRequestHandler.end_headers(self)
+
+    def send_my_headers(self):
+        self.send_header("Cache-Control", "no-cache, no-store, must-revalidate")
```

Comment:
Lint: typically we use single-quotes throughout. Ditto below.

Reply:
Fixed!